.github/workflows/test.yml

```diff
@@ -51,6 +51,20 @@ jobs:
                     MYSQL_ROOT_HOST: "%"
                 ports:
                     - 13306:3306
+            pgsql:
+                image: postgres
+                env:
+                    POSTGRES_USER: testing
+                    POSTGRES_PASSWORD: testing
+                    POSTGRES_DB: testing
+                ports:
+                    - 5432:5432
+                # Set health checks to wait until postgres has started
```

Comment:
Why is this not needed in tests?

Reply:
I don't understand the question... it's explicitly added to the test workflow

Reply:
Sorry, code coverage then

Reply:
It's not technically NEEDED here, but I added it as a bit of a safety net.  I could add it there as well, but it's inconsequential since the first ~100 tests don't actually use the postgres db

Reply:
Hmmm add to both please in that case